thirdeye/thirdeye-frontend/app/pods/services/rootcause-aggregates-cache/service.js

```diff
@@ -68,34 +67,34 @@ export default Service.extend({
 
     // group by metrics and offsets
     const groupedByUrn = [...missing]
-      .map(urn => toAbsoluteUrn(urn, requestContext.compareMode))
-      .map(urn => { return { urn, base: toMetricUrn(urn), offset: urn.split(':')[2] }; })
+      .map((urn) => toAbsoluteUrn(urn, requestContext.compareMode))
+      .map((urn) => {
+        return { urn, base: toMetricUrn(urn), offset: urn.split(':')[2] };
+      })
```

Comment:
Why use two maps function here? One map could finish the result and will increase efficiency.

Reply:
This is also not written by me but updated by prettify.  By looking at it, I believe "toAbsoluteUrn" is doing some work before the next map.

Reply:
If you don't clear the logic, we can refactor when we do the migration.

Reply:
I will condense it into one map.